packages/documentation-v7/src/stories/components/textarea/textarea.stories.ts

```diff
@@ -0,0 +1,262 @@
+      if: {
+        arg: 'floatingLabel',
+        truthy: false,
+      },
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        category: 'General',
+      },
+    },
+    value: {
+      name: 'Value',
+      description: 'The value of the component.',
+      control: {
+        type: 'text',
+      },
+      table: {
+        disable: true,
+      },
+    },
+    size: {
+      name: 'Size',
+      description: "Sets the size of the component's appearance.",
+      if: {
+        arg: 'floatingLabel',
+        truthy: false,
+      },
+      control: {
+        type: 'select',
+        labels: {
+          'form-control-sm': 'Small',
+          'form-control-rg': 'Regular',
+          'null': 'Middle',
+          'form-control-lg': 'Large',
+        },
+      },
+      options: ['form-control-sm', 'form-control-rg', 'null', 'form-control-lg'],
+      table: {
+        category: 'General',
+      },
+    },
+    rows: {
+      name: 'Rows',
+      description:
+        'Attribute to set the initial height, in lines of text, of the `textarea` element.',
+      control: {
+        type: 'number',
+        min: 3,
+        max: 10,
+        step: 1,
+      },
+      table: {
+        category: 'General',
+      },
+    },
+    hint: {
+      name: 'Helper Text',
+      description: 'Text to place in the help text area of the component.',
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'General',
+      },
+    },
+    disabled: {
+      name: 'Disabled',
+      description:
+        'When set to `true`, disables the component\'s functionality and places it in a disabled state.<div className="mt-mini alert alert-info alert-sm">There are accessibility concerns with the disabled state.<br/>Please read our <a href="/?path=/docs/foundations-accessibility--page#disabled-state">disabled state accessibility guide</a>.</div>',
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        category: 'States',
+      },
+    },
+    validation: {
+      name: 'Validation',
+      description: 'Controls the validation state appearance of the component.',
+      control: {
+        type: 'radio',
+        labels: {
+          'null': 'Default',
+          'is-valid': 'Valid',
+          'is-invalid': 'Invalid',
+        },
+      },
+      options: ['null', 'is-valid', 'is-invalid'],
+      table: {
+        category: 'States',
+      },
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+function renderTextarea(args: Args, context: StoryContext) {
+  const [_, updateArgs] = useArgs();
+  const id = `${context.viewMode}_${context.story.replace(/\s/g, '-')}_ExampleTextarea`;
+  const classes = mapClasses({
+    'form-control': true,
+    [args.size]: args.size !== 'null',
+    [args.validation]: args.valiation !== 'null',
```

Comment:
Only write size if args.size is defined and not 'null'.
Same with args.validation.

```suggestion
    [args.size]: args.size && args.size !== 'null',
    [args.validation]: args.validation && args.validation !== 'null',
```